extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -76,12 +73,19 @@
   transform: translateX(calc(100% - var(--option-percentage)))!important;
 }
 
+.i-amphtml-story-interactive-poll-container:not(.i-amphtml-story-interactive-has-data) .i-amphtml-story-interactive-option::after {
+  display: none !important;
+}
+
 .i-amphtml-story-interactive-option-text {
   font-size: 22px !important;
   margin: 0 !important;
   z-index: 1 !important;
   transition: transform var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
   transform-origin: left !important;
+  white-space: nowrap !important;
+  overflow: hidden !important;
+  text-overflow: ellipsis !important;
```

Comment:
As per Hong's guidance, polls should only accept one line of text and truncate with ellipsis